packages/core/src/middleware/koa-audit-log.ts

```diff
@@ -38,7 +38,7 @@ export class LogEntry {
   }
 }
 
-export type LogPayload = Partial<LogContextPayload> & Record<string, unknown>;
+export type LogPayload = Partial<LogContextPayload>;
```

Comment:
i think `& object` would be better to keep the ability of accessing arbitrary keys with known key inference

Reply:
<img width="572" alt="image" src="https://user-images.githubusercontent.com/36393111/211461436-11760d17-3679-4fbc-b3b9-76149d199d60.png">

it is added directly at the `LogContextPayload ` guard level. The `LogContextPayload` will includes the [key:string]: unknown